awkward-cpp/pyproject.toml

```diff
@@ -91,22 +91,22 @@ pytest -p no:cacheprovider \
 """]
 skip = [
     "pp*win*",
-    "pp*-manylinux_i686",
 ]
 test-skip = [
-    "*universal2:arm64",
     "pp*",
     "*musllinux*",
 ]
 build-verbosity = 1
 
 [tool.cibuildwheel.environment]
 PIP_ONLY_BINARY = "cmake,numpy"
-
-[[tool.cibuildwheel.overrides]]
-select = "cp313*"
-environment.PYO3_USE_ABI3_FORWARD_COMPATIBILITY = "1"
+UV_ONLY_BINARY = "cmake,numpy"
```

Comment:
This is probably a good idea, since we use `build[uv]` some of the time, and the variable has this name there.